midi.zig

```diff
@@ -50,7 +52,7 @@ pub const Message = struct {
 
     pub fn channel(message: Message) ?u4 {
         const _kind = message.kind();
-        const _channel = @truncate(u4, message.status);
+        const _channel = @as(u4, @truncate(message.status));
```

Comment:
Aah sorry, I was trying to imply in a previous comment that there are a lot of these places where this needs fixing. I'll just point them out :)
```suggestion
        const _channel: u4 = @truncate(message.status);
```